CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Components/WeatherWidget.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  WeatherWidget.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-10.
+//
+
+import SwiftUI
+
+struct WeatherWidget: View {
+    var forecast: Forecast
+    
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            // MARK: Trapezoid
+            Trapezoid()
+                .fill(Color.weatherWidgetBackground)
+                .frame(width: 342, height: 174)
+            
+            // MARK: Content
+            HStack(alignment: .bottom) {
+                VStack(alignment: .leading, spacing: 8) {
+                    // MARK: Forecast Temperature
```

Comment:
you don't need to use `// MARK:` to split up the views, instead of it create a new variable to explain type of view. 

`//Mark` should be used to split up group of methods or extensions not just small part of views.